timemachine/fe/geometry.py

```diff
@@ -64,15 +114,69 @@ def classify_geometry(mol: Chem.Mol) -> List[LocalGeometry]:
     mol: Chem.Mol
         Input molecule.
 
+    core: List[Int] or None
+        Core indices, if None then all atoms are considered to be in the core.
+
     Returns
     -------
     List[LocalGeometry]
-        List of per atom geometries
+        List of per atom geometries. Dummy atoms are None
+
 
     """
+    if ff is None:
+        ff = Forcefield.load_from_file("smirnoff_1_1_0_sc.py")
```

Comment:
nit: Use DEFAULT_FF ? (https://github.com/proteneer/timemachine/blob/master/timemachine/constants.py#L14)

Reply:
willfix

Reply:
(actually I wanted to use the _sc one to avoid the need to run AM1 calculations since nonbonded interactions are not considered here).